csvqb/csvqb/models/cube/cube.py

```diff
@@ -18,7 +18,6 @@ class CubeMetadata:
     def __init__(self,
                  title: str,
                  uri_safe_identifier: Optional[str] = None,
```

Comment:
Since base_uri isn't what we hoped it was, we've removed it, but that shouldn't be in this PR. But I'll allow it.

Reply:
I removed it :P whilst we were working together. There's no point writing a test for something we're going to remove anyway.

Reply:
@robons I have tried to resolve PR comments and pushed the recent changes. Please, review it. Thank you